src/functions-reference/unbounded_discrete_distributions.Rmd

```diff
@@ -219,29 +219,73 @@ dropping constant additive terms.
 
 ### Stan Functions
 
+<!-- real; neg_binomial_2_log_glm_lpmf; (int y | matrix x, real alpha, vector beta, real phi); -->
+\index{{\tt \bfseries neg\_binomial\_2\_log\_glm\_lpmf }!{\tt (int y \textbar\ matrix x, real alpha, vector beta, real phi): real}|hyperpage}
+
+`real` **`neg_binomial_2_log_glm_lpmf`**`(int y | matrix x, real alpha, vector beta, real phi)`<br>\newline
+The log negative binomial probability mass of `y` given log-location
+`alpha + x * beta` and inverse overdispersion control `phi`, where a
+constant intercept `alpha`, `phi` and dependant variable `y` are used for all observations. 
+The number of columns of `x` needs to match the length of the weight vector `beta`.
+If `x` and `y` are data (not parameters) this function can be executed on a GPU.
+
+<!-- real; neg_binomial_2_log_glm_lpmf; (int y | matrix x, vector alpha, vector beta, real phi); -->
+\index{{\tt \bfseries neg\_binomial\_2\_log\_glm\_lpmf }!{\tt (int y \textbar\ matrix x, vector alpha, vector beta, real phi): real}|hyperpage}
+
+`real` **`neg_binomial_2_log_glm_lpmf`**`(int y | matrix x, vector alpha, vector beta, real phi)`<br>\newline
+The log negative binomial probability mass of `y` given log-location
+`alpha + x * beta` and inverse overdispersion control `phi`, where a
+constant `phi` and independent variable value `y` are used for all observations. Intercept `alpha`
+is used that is allowed to vary with the different observations.
```

Comment:
The intercept `alpha` may vary by observation.

Reply:
fixed